src/main/java/academic/AcademicCommandParser.java

```diff
@@ -48,7 +48,11 @@ public static AcademicCommandType getAcademicCommandType(String command)
         String number = command.substring(command.indexOf("m/") + 2,
                 command.indexOf("e/")).trim();
         String email = command.substring(command.indexOf("e/") + 2);
-        int numberFormatTest = Integer.parseInt(number);
+        int numberAsInteger = Integer.parseInt(number);
+        if (numberAsInteger >= 100000000) {
```

Comment:
I think Number Format Exception will be thrown if your number is greater than 100000000

Reply:
Before I changed this part, number format exception will be thrown when a mobile number that is bigger than 2147483647(max value of int in java). So now I capped it at 100000000 so mobile numbers must be 8 digits and below.

Reply:
okay. got it thanks !